.gitignore

```diff
@@ -22,3 +22,5 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 site_encrypt-*.tar
 
+# Elixir Language Server output
+.elixir_ls
```

Comment:
Personally I have this globally ignored. So in my `~/.gitconfig` I have:

```
[core]
        excludesfile = some_path/.gitignore_global
```

And then in .gitignore_global:

```
.elixir_ls
.vscode
...
```

But I don't see any harm in having this locally ignored.